packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -260,6 +261,13 @@ class WebView extends StatefulWidget {
   /// By default `gestureNavigationEnabled` is false.
   final bool gestureNavigationEnabled;
 
+  /// A Boolean value that determines whether pressing a link displays a preview of the destination for the link.
```

Comment:
This can just say "Whether pressing a link displays [...]", since the fact that it's a boolean value is obvious from the Dart code itself. (I know this was modeled on the comment for the field above, but that's not our usual style so it's better if we don't propagate it.)

Reply:
changed to suggested docs comment, "Whether pressing a link displays ..."